3_javascript/javascript1/week2/javascript.js

```diff
@@ -0,0 +1,146 @@
+//Welcome to my second experience in JAVASCRIPT :)
+
+/* console.log("*******     Flight booking fullname function    *******");
+ function getFullname(firstname, surname) {
+     return `${firstname} ${surname}`;
+}
+
+ const fullname1 = getFullname("Helena", "Hashemi");
+ const fullname2 = getFullname("Shahram", "Shokri");
+
+ console.log(fullname1);
+ console.log(fullname2); */
+
+console.log("*******     Flight booking fullname function / ('Formal fullnam')     *******");
+function getFullname(firstname, surname, gender, userFormalName) {
```

Comment:
You passed a parameter `userFormalName` (and that is perfect) but you are not using it.

Reply:
I couldn't understand what exactly you mean, but tried to correct it